x/claim/types/claim_record_test.go

```diff
@@ -0,0 +1,54 @@
+package types_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/tendermint/spn/testutil/sample"
+	claim "github.com/tendermint/spn/x/claim/types"
+)
+
+func TestClaimRecord_Validate(t *testing.T) {
+	invalidAddress := sample.ClaimRecord(r)
+	invalidAddress.Address = "invalid"
+
+	invalidClaimable := sample.ClaimRecord(r)
+	invalidClaimable.Claimable = sdk.NewInt(-1)
+
+	invalidCompletedMissions := sample.ClaimRecord(r)
+	invalidCompletedMissions.CompletedMissions = []uint64{0, 1, 2, 0}
```

Comment:
Same as for `mission`

I think it's more visual to have the definition in the case table